tests/core/creators/manual/detachedInline.md

```diff
@@ -0,0 +1,37 @@
+@bender-tags: 4.17.0, bug, 4462
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar,link,format,undo,floatingspace,sourcedialog
+
+
+1. Open dev console
+
+2. Click "Toggle" button twice. To hide and show editor.
+
+	**Expected:** Editor shows with same data. Content is editable. Elementspath (at the bottom is updated).
```

Comment:
`Elementspath` plugin is disabled for inline editor, so it should be removed from all `expected` clauses in this test.